docs/source/app-dev/bindings-x-lang/index.rst

```diff
@@ -94,4 +94,13 @@ Links
 
 - gRPC documentation: https://grpc.io/docs/
 
-- DAML Ledger API gRPC protobuf definitions: https://github.com/digital-asset/daml/tree/master/ledger-api/grpc-definitions
+- DAML Ledger API gRPC protobuf definitions
```

Comment:
Should we add this information somewhere in the protobuf tarball, e.g., some readme file?

Reply:
Sure. It's a bit unclear to me, though, which rule originates the tarball published on the GitHub releases page. Do you remember it off the top of your head?

Reply:
I found this in `artifacts.yml` but I'm not 100% sure:
```yaml
- target: //ledger-api/grpc-definitions:ledger-api-protos-tarball
  type: targz
  location:
    groupId: com.daml
    artifactId: ledger-api-protos
```

Reply:
It comes from https://github.com/digital-asset/daml/blob/246d246ee500464f034ce841f0f3942575f1fd0d/release/util.bzl#L86. The line you are showing here should no longer be in `artifacts.yaml` and `artifacts.yaml` is completely unrelated to github releases. It only defines Maven artifacts.

Reply:
Ok, it looks like you're packaging together the contents of `//ledger-api/grpc-definitions:ledger-api-protos.tar.gz` with the DAML-LF stuff. There is probably not much to gain in having it always around, but for simplicity I think I'll just add the `README.md` to the tarball.

Reply:
```
> tar tf bazel-bin/ledger-api/grpc-definitions/ledger-api-protos.tar.gz
./
./com/
...
./com/daml/ledger/api/v1/value.proto
./README.md
```

Reply:
Addressed by https://github.com/digital-asset/daml/pull/5664/commits/9e09c02dd823490a7f35019b1d348ae226bfe395